ui/src/app/components/spinners.tsx

```diff
@@ -1,10 +1,11 @@
 import colors from 'app/styles/colors';
 import {reactStyles} from 'app/utils/index';
+import * as Color from 'color';
 import * as React from 'react';
 
 const styles = reactStyles({
   overlay: {
-    backgroundColor: 'rgba(0, 0, 0, 0)',
+    backgroundColor: 'rgba(200, 200, 200, 0)',
```

Comment:
Could we use `colors.dark` here? I think that would give a similar feel to this.

Reply:
colors.dark is equivalent to rgb(74, 74, 74)... which is pretty far off from this. Closer would be colors.light (rgb(233, 236, 239)) -- I'll go with that, and can follow-up with Lou if / when this starts coming into use.

Reply:
That works for me. I noticed when working with Lou that he tended to use colors.dark with a bunch of whiteness mixed in over colors.light. I suspect that is because if he wanted to hue our backgrounds, he'd want to equivalently hue our overlays and disables?